content/applications/finance/payment_acquirers/stripe.rst

```diff
@@ -171,3 +171,18 @@ listed, you don't have anything to do.
 
 .. seealso::
    - :doc:`../payment_acquirers`
+
+.. _stripe/express-checkout:
+
+Using Express checkout
+======================
```

Comment:
Playing dumb here: what is express checkout? Why is it interesting to configure it?